src/Helmet.js

```diff
@@ -187,7 +187,11 @@ const updateHtmlAttributes = (attributes) => {
         htmlTag.removeAttribute(attributesToRemove[i]);
     }
 
-    htmlTag.setAttribute(HELMET_ATTRIBUTE, helmetAttributes.join(","));
+    if (helmetAttributes.length === attributesToRemove.length) {
+        htmlTag.removeAttribute(HELMET_ATTRIBUTE);
```

Comment:
My intention was to only remove the tracking attribute if there are no attributes coming from Helmet (blank Helmet - clear case).  It's just to keep the DOM clean.  It'll come back if you change to another route with valid htmlAttributes.